config/main.py

```diff
@@ -4153,5 +4151,12 @@ def delete(ctx):
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
 
+
+# Load plugins and register them
+helper = util_base.UtilHelper()
+for plugin in helper.load_plugins(plugins):
+    plugin.register(config)
+
```

Comment:
Would there be a provision for unloading/deregistering ?

Reply:
There is no need. CLI is short living process that I doubt require unregistering, unloading plugin in the middle.